plutus-scb/app/Main.hs

```diff
@@ -274,8 +282,8 @@ runCliCommand config (ForkCommands commands) =
         threads <- traverse forkCommand commands
         waitAny threads
   where
-    forkCommand :: Command -> IO (Async ())
-    forkCommand = async . void . runApp config . runCliCommand config
+    forkCommand ::  Command -> IO (Async ())
+    forkCommand = async . void . runApp LevelDebug config . runCliCommand config
```

Comment:
Should this be passed in from somewhere?

Reply:
Yes and no. 

Really it needs to be passed through to all the mock servers, as well as the core "App" stuff. But that's significantly more replumbing, when my goal was, "Make the commands we expect a user to run less verbose." 

ie. I don't think it really matters if the dev-time stuff logs everything. But the user reports and commands probably shouldn't.

Feel free to disagree though. I don't mind doing the plumbing, it just didn't seem that important.

Reply:
Having said that, I've just pushed a small change to make what's there clearer, at least.